clarity/contracts/pool-token-yield-usda-59760-usda.clar

```diff
@@ -0,0 +1,95 @@
+(impl-trait .trait-sip-010.sip-010-trait)
+(impl-trait .trait-pool-token.pool-token-trait)
+
+(define-fungible-token ayusda-usda)
```

Comment:
the name of the fungible-token does not include all the relevant information (i.e. expiry of yield token and token)